app/templates/components/data-table.hbs

```diff
@@ -1,16 +1,23 @@
 <div data-table class="container-flex--scroll">
```

Comment:
I think this file can simply be removed actually. I seems like the component isn't used in the app anymore?

Reply:
I think you're right.